packages/wxt/src/client/content-scripts/ui/index.ts

```diff
@@ -199,8 +199,25 @@ function getAnchor(options: ContentScriptAnchoredOptions): Element | undefined {
 
   let resolved =
     typeof options.anchor === 'function' ? options.anchor() : options.anchor;
-  if (typeof resolved === 'string')
-    return document.querySelector<Element>(resolved) ?? undefined;
+
+  if (typeof resolved === 'string') {
+    // If the string is an XPath expression (starts with '//' or '/')
+    if (resolved.startsWith('//') || resolved.startsWith('/')) {
```

Comment:
can this be simplified?

```suggestion
    if (resolved.startsWith('/')) {
```

Reply:
Yes, you are correct. It has been resolved.